model/http-common.yaml

```
@@ -52,6 +52,13 @@ groups:
         requirement_level: required
         brief: >
           Port identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to.
+      - ref: http.route
```

```suggestion
      - ref: url.template
```

see the comment below (and discussion in the https://github.com/open-telemetry/semantic-conventions/issues/923)